### PR TITLE
Fix incorrect javascript.

### PR DESCRIPTION
**Pull request by yukimotochern**

### CODE REVIEW

**Comment by cartermp** [comment]
@yukimotochern can you sign the CLA please?

**Comment by svrnm** [comment]
@open-telemetry/javascript-approvers PTAL

**Review by cartermp — APPROVED** [review]
docs-wise changes look good but want JS approval

**Review by pichlermarc — APPROVED** [review]

**Pull request merged**
